[MAINT]: Bump package to fix CVE-2022-23541

### Describe the need

Please upgrade `universal-github-app-jwt` package.
They have recently [fixed CVE related to jsonwebtoken](https://github.com/gr2m/universal-github-app-jwt/pull/66)
![image](https://user-images.githubusercontent.com/10233442/209798518-2b552230-2d6a-482e-801f-ba56d99ec085.png)

[Link to CVE](https://github.com/advisories/GHSA-hjrf-2m68-5959)

### SDK Version

_No response_

### API Version

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

@ebickle for a concrete take, if this is possible, I'm not sure how to fix it. I'm able to update the version in `package.json` to v2.x.x:

```
diff --git a/package.json b/package.json
index 7491b91..0a8ecd6 100644
--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "@types/lru-cache": "^5.1.0",
     "deprecation": "^2.3.1",
     "lru-cache": "^6.0.0",
-    "universal-github-app-jwt": "^1.0.1",
+    "universal-github-app-jwt": "^2.0.1",
     "universal-user-agent": "^6.0.0"
   },

```

and after running `npm i`, see corresponding changes in the package-lock.json:

```
diff --git a/package-lock.json b/package-lock.json
index 85bdd62..2bdfb53 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -17,7 +17,7 @@
         "@types/lru-cache": "^5.1.0",
         "deprecation": "^2.3.1",
         "lru-cache": "^6.0.0",
-        "universal-github-app-jwt": "^1.0.1",
+        "universal-github-app-jwt": "^2.0.1",
         "universal-user-agent": "^6.0.0"
       },
```

However, at this point `npm run build` fails with:

```
 sh$ npm run build

> @octokit/auth-app@0.0.0-development build
> pika-pack build

@pika/pack build v0.3.7
[1/6] Validating source...
[2/6] Preparing pipeline...
      ❇️  pkg/
      » tsconfig.json [compilerOptions.target] should be "ES2020", but found "ES2018". You may encounter problems building.
[3/6] Running @pika/plugin-ts-standard-pkg...
src/get-app-authentication.ts(1,10): error TS2614: Module '"universal-github-app-jwt"' has no exported member 'githubAppJwt'. Did you mean to use 'import githubAppJwt from "universal-github-app-jwt"' instead?
```

Upon making the suggested change, `npm run build` succeeds but `npm run test` fails with three versions of this error:

```
    Details:

    /home/kfcampbell/github/dev/auth-app.js/node_modules/universal-github-app-jwt/index.js:4
    import { getToken } from "#get-token";
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

    > 1 | import githubAppJwt from "universal-github-app-jwt";
        | ^
      2 |
      3 | import { AppAuthentication, State } from "./types";
      4 |

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1449:14)
      at Object.<anonymous> (src/get-app-authentication.ts:1:1)
      at Object.<anonymous> (src/auth.ts:18:1)
      at Object.<anonymous> (src/index.ts:5:1)
      at Object.<anonymous> (test/index.test.ts:6:1)
```

There's some discussion about a backport to v1 to fix the CVE as well over at [universal-github-app-jwt#66](https://github.com/gr2m/universal-github-app-jwt/pull/66).

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "@types/lru-cache": "^5.1.0",
     "deprecation": "^2.3.1",
     "lru-cache": "^6.0.0",
-    "universal-github-app-jwt": "^1.0.1",
+    "universal-github-app-jwt": "^1.1.1",
     "universal-user-agent": "^6.0.0"
   },
   "devDependencies": {

